6/README.md

```diff
@@ -0,0 +1,1078 @@
+---
+domain: rfc.tango-controls.org
+shortname: 6/Database
+name:Database
+status: draft
+editor: 
+contributors: [David Erb](mailto:david.erb@maxiv.lu.se)
+---
+
+This document describes the Tango Database specification.
+
+## Preamble
+
+Copyright (c) 2019 Tango Controls
+
+This Specification is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version. This Specification is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program; if not, see <http://www.gnu.org/licenses>.
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Related RFC specifications (TBD)
+- X/Device Model 
+- X/Attributes 
+- X/Properties 
+
+## Tango Database specification
+
+This specification is intended to formally document the Tango Database. 
+Runtime implementations of the Database in conforming Tango implementations MUST follow this specification.
+
+
+### Goals of the Database
+* provide a means of defining and querying devices.
+* provide means for device servers to be configured from central control
+* serve as nameserver such that the Devices are accessible by name
+
+### Features of the Database
+
+* store configuration data (properties) used at startup of a device server 
+* act as nameserver by storing the dynamic network addresses
+* act as permanent store of dynamic settings which need to be memorized
+* ensure the uniqueness of device name and of aliases
+* keep list of controlled servers which are to be started and run on a particular host
+* limited set functionality provide by file as alternative to network database
+
+### Use Cases of the Database
+
+* register a new device server
+* stores the last user setting point of the current provided by a power supply
+* define persistent configuration properties to be used by a camera every time it starts up
+* allow a subscriber or client to get connect to a stepper motor
+* define properties for all instances of a class of vaccuum pump
+* check if a electrometer is online or not
```

Comment:
Not sure to understand this sentence. The Tango database does not check the availability of the control system devices. It just provides the network ID of the devices.